src/server/WorkerLobbyService.ts

```
@@ -95,7 +95,26 @@ export class WorkerLobbyService {
           publicGameType: gi.publicGameType!,
         } satisfies PublicGameInfo;
       });
-    process.send?.({ type: "lobbyList", lobbies } satisfies WorkerLobbyList);
+
+    const openLobbies = this.gm
+      .openCustomLobbies()
+      .map((g) => g.gameInfo())
+      .filter((gi) => gi.openCustomType != null)
```

ESLint

[error] Expected '!==' and instead saw '!='.

(eqeqeq)